src/base/ClpMap.h

```diff
@@ -106,8 +106,9 @@ class ClpMap
     using EntriesIterator = typename Entries::iterator;
 
     using IndexItem = std::pair<Key, EntriesIterator>;
```

Comment:
This PR makes this type unused. Please either remove it or simplify by renaming IndexItemAllocatorElement back to IndexItem. I recommend the latter. Recall that IndexItem (i.e. map value_type) and IndexAllocatorItem (i.e. allocator value_type) must be the same so we can (continue to) simply use the shorter name for both purposes.

Reply:
Right. Removing.